build/NuSpecs/UmbracoCms.Core.nuspec

```diff
@@ -23,40 +23,23 @@
       <dependency id="AutoMapper" version="[7.0.1,7.999999)" />
       <dependency id="ClientDependency" version="[1.9.6,1.999999)" />
```

Comment:
Is CDF required by the Core proj? If so I'd like to find a way to remove that dependency since it's a Web only type of thing